invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -67,3 +70,19 @@ if (recordExportDownloadDiv) {
     recordExportDownloadDiv
   );
 }
+
+const submitToCommunityModalDiv = document.getElementById("submit-to-community-modal");
+ReactDOM.render(
```

Comment:
discussed in person: this part need to be refactored as a part of another task. We cannot simply import it from react-invenio-deposit becuase deposit components should not be a part of the landing page